legacy_gbdk/gbdk_example/gbt_player.s

```diff
@@ -266,13 +266,17 @@ _gbt_play::
 ;-------------------------------------------------------------------------------
 
 _gbt_pause::
-	lda	hl,2(sp)
+	lda	hl,2(sp) ; (Parameter: 1 = un-pause/resume, 0 = pause)
```

Comment:
Remove this comment, the function prototype already explains how it works, it's better to not duplicate comments.